src/tests/integration/user/userUpdate.spec.ts

```diff
@@ -0,0 +1,295 @@
+
+  it('expects to return a success message and status 200', async () => {
+    const inputParams = {
+      walletId: secondWalletId,
+      firstName: 'Luke',
+      lastName: 'Skywalker',
+      email: `${secondUsername}@pillar.com`,
+      phone: '+44 77 1111 2222',
+      country: 'UK',
+      state: 'CA',
+      city: 'London',
+      tagline: 'Social media consultant',
+      taglineStatus: false,
+      userSearchable: true,
+      betaProgramParticipant: true,
+    };
+
+    Configuration.accessKeys.oAuthTokens.accessToken = secondUserAccessToken;
+
+    const response = await pSdk.user.update(inputParams);
+    expect(response.status).toBe(200);
+    expect(response.data).toEqual({
+      result: 'success',
+      message: 'User was successfully updated',
+      user: {
+        id: expect.any(String),
+        username: `${secondUsername}`,
+        firstName: 'Luke',
+        lastName: 'Skywalker',
+        email: `${secondUsername}@pillar.com`,
+        phone: '+44 77 1111 2222',
+        country: 'UK',
+        state: 'CA',
+        city: 'London',
+        tagline: 'Social media consultant',
+        taglineStatus: false,
+        userSearchable: true,
+        secretId: expect.any(String),
+        betaProgramParticipant: true,
+      },
+    });
+  });
+
+  it('expects to return a conflict error due duplicated email', async () => {
+    const inputParams = {
+      walletId: firstWalletId,
+      firstName: 'Darth',
+      lastName: 'Vader',
+      email: `${secondUsername}@pillar.com`,
+      phone: '+44 77 1111 2222',
+      country: 'UK',
+      state: 'CA',
+      city: 'London',
+      tagline: 'Social media consultant',
+      taglineStatus: false,
+      userSearchable: true,
+      betaProgramParticipant: true,
+    };
+
+    Configuration.accessKeys.oAuthTokens.accessToken = firstUserAccessToken;
+
+    try {
+      await pSdk.user.update(inputParams);
+    } catch (error) {
+      expect(error.response.status).toEqual(409);
+      expect(error.response.data.message).toEqual(errEmailDuplicated.message);
+    }
+  });
+
+  it('first user deletes email and now second user can use his email', async () => {
+    let inputParams = {
+      walletId: secondWalletId,
+      email: '',
+    };
+
+    Configuration.accessKeys.oAuthTokens.accessToken = secondUserAccessToken;
+    await pSdk.user.update(inputParams);
+
+    Configuration.accessKeys.oAuthTokens.accessToken = firstUserAccessToken;
+
+    inputParams = {
+      walletId: firstWalletId,
+      email: `${secondUsername}@pillar.com`,
+    };
+
+    const response = await pSdk.user.update(inputParams);
+    expect(response.status).toBe(200);
+    expect(response.data).toEqual({
+      result: 'success',
+      message: 'User was successfully updated',
+      user: {
+        id: expect.any(String),
+        email: `${secondUsername}@pillar.com`,
+        secretId: expect.any(String),
+        username: firstUsername,
+      },
+    });
+  });
+
+  if (env === 'test') {
```

Comment:
What do we need this for? :thinking:

Reply:
Because sometimes we change the env to test against a local instance of platform core, and some errors are not happening, for example a 500 error, so we put those tests only for the env = test